lib/trailblazer/activity/path.rb

```diff
@@ -81,12 +81,27 @@ def sequence_insert_options
           }
         end
 
+        def normalize_duplications((ctx, flow_options), *)
+          return Right, [ctx, flow_options] if ctx[:replace]
+
```

Comment:
Can't we introduce two steps for this, on the `Normalizer` level?

Reply:
We could, but I wanted to
- Avoid duplicating the `if ctx[:replace]` check in other place(s)
- Have duplication avoidance strategies under one normalizer method.